src/python/grpcio/tests/unit/test_common.py

```diff
@@ -61,6 +61,10 @@ def metadata_transmitted(original_metadata, transmitted_metadata):
   original = collections.defaultdict(list)
   for key_value_pair in original_metadata:
     key, value = tuple(key_value_pair)
+    if not isinstance(key, bytes):
```

Comment:
The only thing I found that looked applicable was ```six.b()```, but that attempts to encode all inputs, which fails if the input is already a byte string.